src/groups/mqb/mqbblp/mqbblp_storagemanager.cpp

```diff
@@ -1510,6 +1510,17 @@ void StorageManager::stop()
                              bdlf::PlaceHolders::_2));  // latch
 }
 
+void StorageManager::initializeQueueKeyInfoMap(
+    BSLS_ANNOTATION_UNUSED const mqbc::ClusterState* clusterState)
+{
+    // executed by cluster *DISPATCHER* thread
+
+    // PRECONDITION
+    BSLS_ASSERT_SAFE(d_dispatcher_p->inDispatcherThread(d_cluster_p));
+
+    BSLS_ASSERT_OPT(false && "This method should only be invoked in FSM mode");
```

Comment:
Is this really what we want?  This pair of asserts seems wrong...the second one can fire even if we are in the dispatcher thread?

Reply:
The only caller of `initializeQueueKeyInfoMap()` is [`mqbc::ClusterStateManager`](https://github.com/kaikulimu/blazingmq/blob/f3d03d74c4d6bb335080b1d4120d58bcaebe5fc3/src/groups/mqb/mqbc/mqbc_clusterstatemanager.cpp#L271), and it should always use `mqbc` version of `StorageManager`. By design, this method should only be invoked in FSM mode. If you can think of a better way to express this, please let me know.

Reply:
I see.  In this case, the `false` makes sense.  Perhaps, "Only the FSM version of this method from `mqbc::StorageManager` should be invoked." as the string?

Reply:
Agreed. Will change

Reply:
d